src/storekit.cs

```diff
@@ -393,12 +419,14 @@ interface SKRequestDelegate {
 	[Mac (10,9)]
 	[BaseType (typeof (SKRequest))]
```

Comment:
As @spouliot mentioned with others, move the [Watch (6,2)] to the top, just need to add one and the wraps should not need any attr decoration.